shared/lib/sharing/action-types.js

```diff
@@ -0,0 +1,3 @@
+export const FETCH_PUBLICIZE_CONNECTIONS = 'FETCH_PUBLICIZE_CONNECTIONS';
+export const RECEIVE_PUBLICIZE_CONNECTIONS = 'RECEIVE_PUBLICIZE_CONNECTIONS';
+export const FAIL_PUBLICIZE_CONNECTIONS_REQUEST = 'FAIL_PUBLICIZE_CONNECTIONS_REQUEST';
```

Comment:
>Would you later consider something like `keyMirror` here?

I'm not sure. It doesn't seem to provide a huge benefit, aside from mitigating the chance for a typo in assigning the value. We've been bitten enough by `react/lib` deprecations (`classSet`, `merge`, `joinClasses`) that I'm a bit wary of using anything there.